### PR TITLE
Chocolatey Package Search Command 

### LINKED ISSUE #1047: Chocolatey Package Search Command
> Chocolatey is software management automation for Windows that wraps installers, executables, zips, and scripts into compiled packages.

https://chocolatey.org/packages

### PR DESCRIPTION
**Pull request by almeidx**
Closes #1047

### CODE REVIEW

File: src/commands/misc/chocolatey.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  searchResultFormatter (i) {
+    return `[${i.Title} - ${i.Authors}](${i.GalleryDetailsUrl})`
+  }
+
+  async handleResult ({ t, author, channel, language }, { GalleryDetailsUrl, IconUrl, Title, Summary }) {
+    channel.send(
+      new SwitchbladeEmbed(author)
+        .setColor(this.embedColor)
+        .setAuthor(t('commmands:chocolatey.chocolatey'), this.embedLogoURL, 'https://chocolatey.org/packages')
```
Comment (Psykka) [inline src/commands/misc/chocolatey.js:28]: you typed commands with three "m"

**Review by Doges — COMMENTED** [review]
btw you forgot the commands.json

**Comment by almeidx** [comment]
> btw you forgot the commands.json

it's still WIP

**Comment by Doges** [comment]
> 
> 
> > btw you forgot the commands.json
> 
> it's still WIP

that's why i didn't reviewed it yet

File: src/commands/misc/chocolatey.js (hunk 1 above)
Comment (pedrofracassi) [reply src/commands/misc/chocolatey.js]: hmmm

Comment (almeidx) [reply src/commands/misc/chocolatey.js]: fixed in https://github.com/SwitchbladeBot/switchblade/pull/1166/commits/3529edfd90dce302ae077dbf033a413ed4aaaae7

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1663,5 +1663,12 @@
     "verified": "Verified account",
     "yes": "Yes",
     "no": "No"
+  },
+  "chocolatey": {
+    "commandDescription": "Searches for a packages on Chocolatey",
+    "commandUsage": "<package name>",
+    "by": "by {{- authors }}",
+    "installs": "{{- count }} installs",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1671]:
```suggestion
    "by": "by {{authors}}",
    "installs": "{{count}} installs",
```

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -1663,5 +1663,12 @@
     "verified": "Verified account",
     "yes": "Yes",
     "no": "No"
+  },
+  "chocolatey": {
+    "commandDescription": "Searches for a packages on Chocolatey",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1668]:
```suggestion
    "commandDescription": "Searches for packages on Chocolatey",
```

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

**Review by pedrofracassi — DISMISSED** [review]

File: src/apis/Chocolatey.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+module.exports = class Chocolatey extends APIWrapper {
+  constructor () {
+    super({
+      name: 'chocolatey'
+    })
+  }
+
+  async search (query) {
+    return axios.get(`https://chocolatey.org/api/v2/Search()?$filter=IsLatestVersion&$skip=0&$top=10&searchTerm=${encodeURIComponent(`'${query}'`)}&targetFramework=%27%27&includePrerelease=false`)
```
Comment (davipatury) [inline src/apis/Chocolatey.js:12]: Use the `params` options to make the request more readable.

**Review by davipatury — CHANGES_REQUESTED** [review]

File: src/apis/Chocolatey.js (hunk 4 above)
Comment (almeidx) [reply src/apis/Chocolatey.js]: https://github.com/SwitchbladeBot/switchblade/pull/1166/commits/bce310460b66e08b115588316de7fab69bd7b1e7

**Review by pedrofracassi — APPROVED** [review]
LGTM!

**Comment by Doges** [comment]
you know what I'm gonna say

**Comment by metehus** [comment]
image

**Review by metehus — APPROVED** [review]
LNTM

**Review by jaxxibae — APPROVED** [review]
LGTM

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**